poc_mobile_verifier/src/heartbeats.rs

```diff
@@ -0,0 +1,51 @@
+//! Heartbeat storage
+
+use chrono::{DateTime, NaiveDateTime, Utc};
+use futures::stream::TryStreamExt;
+use helium_crypto::PublicKey;
+use rust_decimal::Decimal;
+use sqlx::{Postgres, Transaction};
+use std::collections::HashMap;
+
+#[derive(sqlx::FromRow)]
+pub struct Heartbeat {
+    pub id: PublicKey,
+    pub weight: Decimal,
+    pub timestamp: NaiveDateTime,
```

Comment:
this seems redundant to the `HeartbeatValue` struct below; are we not just making this struct `Heartbeat { pub id: PublicKey, weight: HearbeatValue }` because it wouldn't convert cleanly to a sql row?